assets/js/checkout.js

```diff
@@ -184,109 +190,87 @@ $(function () {
 
     switch (checkoutOptions.subscription_type) {
       case 'aws':
-        total = subtotal = pricing.subscriptions.aws.price.value;
+        monthlyTotal = pricing.subscriptions.aws.price.value;
         break;
       case 'gcp':
-        total = subtotal = pricing.subscriptions.gcp.price.value;
+        monthlyTotal = pricing.subscriptions.gcp.price.value;
         break;
       default: // do nothing
     }
 
     if (checkoutOptions.pro_support) {
       switch (checkoutOptions.subscription_type) {
         case 'aws':
-          total += subtotal = pricing.subscriptions.aws.pro_support_price.value;
+          monthlyTotal += pricing.subscriptions.aws.pro_support_price.value;
           break;
         case 'gcp':
-          total += subtotal = pricing.subscriptions.gcp.pro_support_price.value;
+          monthlyTotal += pricing.subscriptions.gcp.pro_support_price.value;
           break;
         default: // do nothing
       }
     }
 
+    // CIS Compliance is only on AWS for now
+    if (checkoutOptions.setup_compliance) {
+      monthlyTotal += pricing.subscriptions.aws.cis_compliance_price.value;
+    }
+
+    dueNowTotal = monthlyTotal;
+
     // Only AWS supports the Ref Arch
     if (checkoutOptions.setup_deployment) {
-      subscriptionTotal = subtotal;
-      //subtotal += 4950;
+      dueNowTotal += 4950;
+      $('#due-monthly-block').show();
+    } else {
+      // only show the monthly disclaimer when it differs from due now
+      $('#due-monthly-block').hide();
     }
 
-    // CIS Compliance is only on AWS for now
-    if (checkoutOptions.setup_compliance) {
-      total += subtotal = pricing.subscriptions.aws.cis_compliance_price.value;
+    $('#due-now').text(dueNowTotal.toLocaleString());
+    $('.monthly-total').text(monthlyTotal.toLocaleString());
+  }
+
+  // Update the Recurly checkout link URL
+  function _updateCheckoutLink() {
+    const type = checkoutOptions.subscription_type;
+    const support = checkoutOptions.pro_support;
+    const setup = checkoutOptions.setup_deployment;
+    const compliance = checkoutOptions.setup_compliance;
+
+    var href = "https://gruntwork-sandbox.recurly.com/subscribe/" + type + "-monthly?";
+
+    const addOns = { "users": 20 };
+    if (support) {
+      addOns["pro-support"] = 1;
+    }
+    if (setup) {
+      addOns["ref-arch"] = 1;
+    }
+    if (compliance) {
+      addOns["cis-aws-foundations"] = 1;
     }
 
-    $('#subscription-price').text(total.toLocaleString());
-    $('#subscription-subtotal').text(subtotal.toLocaleString());
+    const params = {
+      theme: "modern",
+      add_on_code: Object.keys(addOns).toString(),
+      add_on_quantity: Object.values(addOns).toString(),
+    };
 
-    _deferCheckout(checkoutOptions.subscription_type,
-      checkoutOptions.pro_support,
-      checkoutOptions.setup_deployment,
-      checkoutOptions.setup_compliance);
-  }
+    for (const key in params) {
+      href += "&" + key + "=" + params[key]
```

Comment:
Does this not end up generating a url like: `"......-monthly?&users=20"`? Note the `&` after the `?`

Reply:
It does. It's avoidable, but makes the code a little uglier. (I actually handled it at first, then took it out to simplify the logic.) It doesn't impact the behavior to begin with the separator — does it explicitly go against the spec or fail to work in some contexts? I'm not sure.

Reply:
Hmm it does produce such a URL: 
![image](https://user-images.githubusercontent.com/34349331/134400686-6fd1bbe3-6b6b-4e75-8cf7-5bf0b87adf13.png)

Usually you just see `?firstParam=value&nextParam=anothervalue.....` Not sure if that's a legit problem or just a technicality

Reply:
Yeah, exactly. It works fine in my experience, but I agree it's slightly unconventional. Would this satisfy you more? It's no big deal to add it, so maybe we should just conform to convention.
```
    sep = "";
    for (const key in params) {
      href += sep + key + "=" + params[key];
      sep = "?";
    }
```

Reply:
Pushed this change!